ethdb/interface.go

```diff
@@ -16,7 +16,7 @@
 
 package ethdb
 
-// Code using batches should try to add this much data to the batch.
+// IdealBatchSize is the amount of data vode using batches should try to add to the batch.
```

Comment:
`// IdealBatchSize is the amount of data batches should accumulate before flushing.`